src/System Application/App/Rest Client/src/Authentication/HttpAuthOAuthClientCredentials.Codeunit.al

```diff
@@ -49,7 +49,7 @@ codeunit 2361 "HttpAuthOAuthClientCredentials" implements "Http Authentication"
     [NonDebuggable]
```

Comment:
@darjoo we should be able to remove the `[NonDebuggable]` property from those procedures now.

Reply:
Yeah absolutely.